metomi/rose/loc_handlers/rsync.py

```diff
@@ -24,6 +25,41 @@
 from metomi.rose.popen import RosePopenError
 
 
+class PreRsyncCheckError(Exception):
+    """Error to raise if the checks"""
```

Comment:
if the checks...?

Reply:
...throw the papal envoy out of a window?

Will make this make sense.